WW_FullyLeptonic/WW_FullyLeptonic.lua

```diff
@@ -0,0 +1,136 @@
+-- Load the library containing the matrix element
+load_modules('../WW_FullyLeptonic/pp_WW_fully_leptonic/build/libme_WW_ee.so')
+
+-- Global parameters used by several modules
+-- Changing these has NO impact on the value of the parameters used by the matrix element!
+parameters = {
+    energy = 13000.,
+    W_mass = 80.419002,
+    W_width = 2.047600,
+}
+
+-- Configuration of Cuba
+cuba = {
+    relative_accuracy = 0.05,
+    verbosity = 3
+}
+
+-- The transfer functions take as input the particles passed in the computeWeights() function,
+-- and each add a dimension of integration
+GaussianTransferFunction.tf_p1 = {
+    -- getpspoint() generates an input tag of type `cuba::ps_points/i`
+    -- where `i` is automatically incremented each time the function is called.
+    ps_point = getpspoint(),
+    reco_particle = 'input::particles/1',
+    sigma = 0.05,
+}
+
+GaussianTransferFunction.tf_p2 = {
+    ps_point = getpspoint(),
+    reco_particle = 'input::particles/2',
+    sigma = 0.10,
+}
+
+inputs_before_perm = {
+    'tf_p1::output',
+    'tf_p2::output',
+}
+
+
+inputs = inputs_before_perm
```

Comment:
Since here you have no permutation, you can remove this line and just rename the above `inputs`